core/ustring.cpp

```diff
@@ -2776,14 +2776,14 @@ String String::format(const Variant &values, String placeholder) const {
 					Variant v_key = value_arr[0];
 					String key;
 
-					key = v_key.get_construct_string();
+					key = v_key;
```

Comment:
Do you know if there's a specific reason to do this in two statements instead of just `String key = v_key;`?

Reply:
I don't think there is one. I just left it the way it was.
Might combine the lines as well.